README.md

```diff
@@ -136,7 +136,7 @@ BUNDLE_GEMFILE=/path/to/your/project/.ruby-lsp/Gemfile bundle install
 ```
 
 If after these steps the Ruby LSP is still not initializing properly, please report the issue
-[here](https://github.com/Shopify/vscode-ruby-lsp/issues/new).
+[here](https://github.com/Shopify/vscode-ruby-lsp/issues/new?assignees=&labels=bug&template=bug_template.yml).
```

Comment:
Do we need the assignees part? I think we can remove it. Other than that, looks good to me!